zookeeper-server/src/main/java/org/apache/zookeeper/KeeperException.java

```diff
@@ -407,6 +411,10 @@ public void setCode(int code) {
          *  but client is not configured with SASL authentication or configuted with SASL but failed
          *  (i.e. wrong credential used.). */
         SESSIONCLOSEDREQUIRESASLAUTH(-124),
+        /** Exceeded the count quota that was set on the path.*/
+        COUNTQUOTAEXCEEDED(-125),
+        /** Exceeded the bytes quota that was set on the path.*/
+        BYTESQUOTAEXCEEDED(-126),
```

Comment:
I feel that for the client it does not have much value to have two different error codes.
The fact is that "you are using too much resources".

Having two codes would make sense only if the application could automatically handle the error